envs/example/defaults.yml

```diff
@@ -336,7 +345,12 @@ keystone:
       public_url: https://{{ endpoints.main }}:6384
       internal_url: https://{{ endpoints.main }}:6384
       admin_url: https://{{ endpoints.main }}:6384
-
+    - name: magnum
+      type: container
+      description: 'Magnum Container Service'
+      public_url: https://{{ endpoints.main }}:9511
+      internal_url: https://{{ endpoints.main }}:9511
+      admin_url: https://{{ endpoints.main }}:9511
```

Comment:
Are these the default ports that the service normally runs on, or our silly different port thing? It should be the standard upstream port to align with our future.